games-engines/openmw/files/openmw-0.47.0-mygui-license.patch

```diff
@@ -4,13 +4,13 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index d69352c94..5f97c86e9 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -470,9 +470,6 @@ IF(NOT WIN32 AND NOT APPLE)
-     #    INSTALL(PROGRAMS "${OpenMW_BINARY_DIR}/Plugin_MyGUI_OpenMW_Resources.so" DESTINATION "${LIBDIR}" )
-     #ENDIF(BUILD_MYGUI_PLUGIN)
+@@ -856,9 +856,6 @@ elseif(NOT APPLE)
+             INSTALL(PROGRAMS "${INSTALL_SOURCE}/openmw-wizard" DESTINATION "${BINDIR}" )
+         ENDIF(BUILD_WIZARD)
```

Comment:
I'm not familiar with openmw, but is this patch required to fix the bug we have in hand?

Reply:
This patch is for 9999 version, and currently it doesn't build with the old version of the patch. It's not related to the bug.

Reply:
I've moved it to a separate commit